trustee-on-azure/01-infrastructure-setup-Azure/02-managing-vm-instances-azure.md

```diff
@@ -0,0 +1,208 @@
+# Managing VM Instances and Images on Azure
+
+**Authors:** demeng@redhat.com \
+**Jira Issues:** 
+- [VIRTWINKVM-1122](https://issues.redhat.com/browse/VIRTWINKVM-1122)
+- [COCL-40](https://issues.redhat.com/browse/COCL-40)
+
+**References:**
+- [VM image for Azure](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/src/cloud-api-adaptor/azure/build-image.md)
+- [Provisioning Fedora CoreOS on Azure](https://docs.fedoraproject.org/en-US/fedora-coreos/provisioning-azure/)
+- [Azure: Launch a confidential VM](https://github.com/coreos/fedora-coreos-docs/pull/671/files)
+- [Install Azure CLI on Linux](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-linux)
+
+## Step 1: Install the Azure CLI
+```bash
+# RHEL9 host could prefer the following instructions, other versions of host could refer to the above link.
+mkdir Azure; cd Azure/
+sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc
+sudo dnf install -y https://packages.microsoft.com/config/rhel/9.0/packages-microsoft-prod.rpm
+sudo dnf install azure-cli
+```
+
+## Step 2: Upload Image Version to Azure
```

Comment:
We have the official coreos documentation which described these steps, instead of replicate the documentation, you could simply reference it: https://docs.fedoraproject.org/en-US/fedora-coreos/provisioning-azure/ . Otherwise, we risk to remain outdated